docs/source/workflow/requests.md

```diff
@@ -93,6 +93,32 @@ For more details, see [the CSRF prevention documentation](../security/cors#preve
 
 Incremental delivery is a [Stage 2: Draft Proposal](https://github.com/graphql/graphql-spec/pull/742) to the GraphQL specification which adds `@defer` and `@stream` executable directives. These directives allow clients to specify that parts of an operation can be sent after an initial response, so that slower fields do not delay all other fields. As of September 2022, the `graphql` library (also known as `graphql-js`) upon which Apollo Server is built implements incremental delivery only in the unreleased major version 17. If a pre-release of `graphql@17` is installed in your server, Apollo Server 4 can execute these incremental delivery directives and provide streaming [`multipart/mixed`](https://github.com/graphql/graphql-over-http/blob/main/rfcs/IncrementalDelivery.md) responses.
 
+Support for incremental delivery in graphql version 17 is [opt-in](https://github.com/robrichard/defer-stream-wg/discussions/12), meaning the directives are not defined by default. In order to use `@defer` or `@stream`, you must provide the definitions for them in your SDL like so:
```

Comment:
Is it worth noting that you only have to add one of them if you only want to use one? idk.

Reply:
How's this? [23eb15a](https://github.com/apollographql/apollo-server/pull/7281/commits/23eb15a3b91623c6310d09679e493f7dab1a5c74)

Reply:
Sure!